wpimath/src/test/java/edu/wpi/first/math/trajectory/ExponentialProfileTest.java

```diff
@@ -4,19 +4,21 @@
 
 package edu.wpi.first.math.trajectory;
 
+import static edu.wpi.first.units.Units.RadiansPerSecond;
 import static org.junit.jupiter.api.Assertions.assertAll;
 import static org.junit.jupiter.api.Assertions.assertEquals;
 import static org.junit.jupiter.api.Assertions.assertNotEquals;
 import static org.junit.jupiter.api.Assertions.assertTrue;
 
 import edu.wpi.first.math.controller.SimpleMotorFeedforward;
+import edu.wpi.first.units.MutableMeasure;
 import java.util.List;
 import org.junit.jupiter.api.Test;
 
 class ExponentialProfileTest {
   private static final double kDt = 0.01;
   private static final SimpleMotorFeedforward feedforward =
-      new SimpleMotorFeedforward(0, 2.5629, 0.43277);
+      new SimpleMotorFeedforward(0, 2.5629, 0.43277, 0.01);
```

Comment:
Use the kDt constant instead of hardcoding it.

Reply:
oof